common/Dockerfile.common

```diff
@@ -42,6 +42,9 @@ COPY run_scarab_mode_4_allbench.sh /usr/local/bin/run_scarab_mode_4_allbench.sh
 COPY gather_cluster_results.py /usr/local/bin/gather_cluster_results.py
 COPY ./common/common_entrypoint.sh /usr/local/bin/common_entrypoint.sh
 
+COPY run_single_simpoint.sh /usr/local/bin/run_single_simpoint.sh
+RUN chmod +x /usr/local/bin/run_single_simpoint.sh
```

Comment:
Not sure why this didn't work for me.
Just do this outside of the docker container and update scripts worked.

Reply:
Did you use a pre-existing container? I added these commands to `update_scripts.sh` as well to fix this issue. I'll document this in the ReadMe

Reply:
I tested from the very beginning (docker build)

Reply:
I added a line to the `slurm_runner.py` script to run `chmod +x /usr/local/bin/run_single_simpoint.sh` before a simpoint is run to fix this issue